zerver/models.py

```diff
@@ -173,7 +173,7 @@ class Realm(models.Model):
     INVITES_STANDARD_REALM_DAILY_MAX = 3000
     MESSAGE_VISIBILITY_LIMITED = 10000
     AUTHENTICATION_FLAGS = ['Google', 'Email', 'GitHub', 'LDAP', 'Dev',
-                            'RemoteUser', 'AzureAD', 'SAML', 'GitLab', 'Apple']
+                            'RemoteUser', 'AzureAD', 'SAML', 'GitLab', 'Apple', 'Cognito']
```

Comment:
This is a particularly important place to have "AWS Cognito" - though we should stick to this convention everywhere, and never have "Cognito" appearing alone - including backend class name (`AWSCognitoBackend` would be a better name for example) etc.

Reply:
I will have a look, but python social auth plugin refers to it as Cognito so some places it will have to stay Cognito.